src/sentry/static/sentry/app/components/footer.tsx

```diff
@@ -41,16 +31,40 @@ function Footer() {
                 value: config.version.build.substring(0, 7),
               })}
             </Build>
-          </div>
+          </React.Fragment>
         )}
-        <LogoLink />
-        <Hook name="footer" />
       </div>
+      <LogoLink />
+      <Links>
+        <FooterLink className="hidden-xs" href="/api/">
```

Comment:
Do we need these `hidden-xs` classes even though we are hiding the entire footer on `breakpoints[0]`?